### PR TITLE
TKO 64: BE: Create Users Table

### PR DESCRIPTION
**Pull request by nixy97**
# Description
Added first_name and last_name attributes to the Users table
  
# Testing
run php artisan migrate:fresh to see the new attributes

# Installation
'Are there any dependencies the reviewer needs to download?'

### CODE REVIEW

**Review by MikkalMc — COMMENTED** [review]
We need to remove the 'name' field within this model and just stick to first_name and last_name

**Review by MikkalMc — CHANGES_REQUESTED** [review]
Remove name field

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 1]
```diff
@@ -16,7 +16,9 @@ public function up()
         Schema::create('users', function (Blueprint $table) {
             $table->increments('id');
             $table->string('name');
```
Comment (MikkalMc) [inline database/migrations/2014_10_12_000000_create_users_table.php]: Remove the name field in migration plozz,

File: database/migrations/2014_10_12_000000_create_users_table.php [hunk 2]
```diff
@@ -16,7 +16,9 @@ public function up()
...
-            $table->string('email')->unique();
+            $table->string('first_name')->nullable();
```
Comment (MikkalMc) [inline database/migrations/2014_10_12_000000_create_users_table.php:18]: by introducing these, we will also need to update the registration service :+1:

**Review by MikkalMc — CHANGES_REQUESTED** [review]

**Review by alecnicolas — APPROVED** [review]
Good, just make Mikkal's changes

**Review by MikkalMc — APPROVED** [review]
cool beans

**Pull request merged**